src/main/java/ui/GUIController.java

```diff
@@ -102,11 +117,43 @@ public void panelFilterExpressionChanged(FilterPanel changedPanel) {
                 .collect(Collectors.toList());
     }
 
-    private void updateAPIBox(UpdateRateLimitsEvent e) {
-        Platform.runLater(() -> apiBox.setText(String.format("%s/%s",
-                                                             e.remainingRequests,
-                                                             Utility.minutesFromNow(e.nextRefreshInMillisecs)))
-        );
+    private void updateRemainingRateEvent(UpdateRateLimitsEvent e) {
```

Comment:
The method name can be improved. `RemainingRate` --> `RefreshRate`? Remove `Event` from the name?